html/audiobridgetest.js

```diff
@@ -284,7 +284,7 @@ $(document).ready(function() {
 									var addButtons = false;
 									if($('#roomaudio').length === 0) {
 										addButtons = true;
-										$('#mixedaudio').append('<audio class="rounded centered" id="roomaudio" width="100%" height="100%" autoplay/>');
+										$('#mixedaudio').append('<audio class="rounded centered" id="roomaudio" width="100%" height="100%" autoplay playsinline/>');
```

Comment:
Is this an audio-only tag? If so, I don't think playsinline is needed, it's only for video elements.

Reply:
Ack, thx for the clarification! Just removed it.